pkg/audit/manager.go

```diff
@@ -280,17 +259,28 @@ func (am *Manager) audit(ctx context.Context) error {
 	timestamp := startTime.UTC().Format(time.RFC3339)
 	am.log = log.WithValues(logging.AuditID, timestamp)
 	logStart(am.log)
+	exportErrorMap := make(map[string]error)
+	if err := am.exportSystem.Publish(context.Background(), *auditConnection, *auditChannel, exportutil.ExportMsg{Message: "audit is started", ID: timestamp}); err != nil {
+		exportErrorMap[err.Error()] = err
+		am.log.Error(err, "failed to export audit start message")
+	}
 	// record audit latency
 	defer func() {
-		logFinish(am.log)
 		endTime := time.Now()
 		latency := endTime.Sub(startTime)
+		logFinish(am.log, latency)
 		if err := am.reporter.reportLatency(latency); err != nil {
 			am.log.Error(err, "failed to report latency")
 		}
 		if err := am.reporter.reportRunEnd(endTime); err != nil {
 			am.log.Error(err, "failed to report run end time")
 		}
+		if err := am.exportSystem.Publish(context.Background(), *auditConnection, *auditChannel, exportutil.ExportMsg{Message: "audit is completed", ID: timestamp}); err != nil {
+			exportErrorMap[err.Error()] = err
+		}
+		for _, v := range exportErrorMap {
+			am.log.Error(v, "failed to export audit violation")
```

Comment:
Are these intended to be posted anywhere (i.e. configmap/crd/constraint) or just logged for now?

Reply:
Once we have a CRD in place, these may be reported there. We still need to decide what kinds of errors belong on constraint vs connection config. In short, these errors will be reported somewhere.